packages/odyssey-storybook/src/installation/Odyssey-React-MUI.stories.mdx

```diff
@@ -1,7 +1,7 @@
 import { Meta } from "@storybook/addon-docs/blocks";
-
-import Readme from "@okta/odyssey-react-mui/README.md";
+import Markdown from "markdown-to-jsx";
+import Readme from "@okta/odyssey-react-mui/README.md?raw";
```

Comment:
requires "./README.md" to be to exported from `packages/odysssey-react-mui/package.json`